piper/src/main/scala/molmed/qscripts/GenotypeConcordance.scala

```diff
@@ -0,0 +1,110 @@
+  var snpGenotypes: Seq[File] = Nil
+
+  @Input(doc = "one or more files with variant calls to be checked against the genotypes - the eval set", fullName = "vcffile", shortName = "vcf", required = false)
+  var variantCalls: Seq[File] = Nil
+
+  @Input(doc = "one or more bam files to call variants for and check against the genotypes - the eval set", fullName = "bamfile", shortName = "bam", required = false)
+  var bamfiles: Seq[File] = Nil
+
+  @Argument(doc = "directory where all output will be written", fullName = "outputdir", shortName = "out", required = false)
+  var outputDir: File = _
+
+  @Argument(doc = "project id for the analysis", fullName = "projectid", shortName = "pid", required = false)
+  var projectId: String = UppmaxConfig.defaultProjectId
+
+  @Input(doc = "genome reference file in fasta format", fullName = "reference", shortName = "ref", required = true)
+  var reference: File = _
+
+  @Argument(doc = "Number of threads to use by default", fullName = "number_of_threads", shortName = "nt", required = false)
+  var nbrOfThreads: Int = 1
+
+  @Argument(doc = "File containing license key for disabling GATK phone home feature", fullName = "gatk_key", shortName = "gatkKey", required = false)
+  var gatkKey: File = _
+
+  /**
+    * **************************************************************************
+    * Main script
+    * **************************************************************************
+    */
+
+  def script() {
+
+    // verify that we got something to work with
+    if ((variantCalls != Nil && bamfiles != Nil) || (variantCalls == Nil && bamfiles == Nil)) {
+      throw new IllegalArgumentException("Either variant VCF file(s) or mapped BAM file(s) must be specified")
+    }
+
+    // setup configs and utilities
+    val gatkOptions =
+      GATKConfig(reference, nbrOfThreads, 1, None, None, None, gatkKey = Option(gatkKey))
+    val variantCallingUtils = new VariantCallingUtils(gatkOptions, projectName = Some(projectId), UppmaxConfig())
+    val variantCallingConfig = VariantCallingConfig(
+      qscript = this,
+      bams = bamfiles,
+      outputDir = outputDir.getOrElse(new File("genotype_concordance")),
+      bcftoolsPath = bcftoolsPath,
+      runSeparatly = false,
+      isExome = false,
+      isLowPass = false,
+      noRecal = true,
+      noIndels = true,
+      testMode = false,
+      skipAnnotation = true,
+      skipVcfCompression = false,
+      noBAQ = false)
+
+    // combine the comp files into one vcf
+    val compVcf = variantCallingUtils.combineVcfFiles(
+      variantCallingConfig, snpGenotypes, new File(projectId + "." + String.valueOf(System.currentTimeMillis()/1000) + ".genotypes.combined.vcf.gz"))
+
+    // if eval VCF files were specified as input, do concordance check on those,
+    // else call variants on the bam files and check them
+    if (variantCalls != Nil) {
```

Comment:
yeah, thanks. I wasn't sure how to handle the Queue/GATK options. I may leave it like this for now so if you'd merge, I'll be happy!